apps/events-helsinki/src/domain/search/eventSearch/constants.tsx

```diff
@@ -96,28 +96,28 @@ export enum EVENT_SEARCH_FILTERS {
 export const CATEGORY_CATALOG = {
   [EventTypeId.Course]: {
```

Comment:
Should this have `EventTypeId.Event` ?

Reply:
This is changed in a later commit to use EventTypeId.General as in events-helsinki-ui repository.